analysis/01_meetup_api_discovery/new_contributor_onboarding/JulianaLam/Question.rtf

```diff
@@ -34,4 +34,10 @@ Pool a sample of cities with high, moderate, and low GDP.\
 1. GDP of sample city\
 2. Number of meet-up events/month\
 3. Mean and SD of RSVP\'92s/meet-up\
-4. Proportion of RSVP\'92s/city population}
\ No newline at end of file
+4. Proportion of RSVP\'92s/city population\
+\
+
+\b Notes:\
+
+\b0 Use of real GDP in Chained Dollars to account for inflation and anomalies. \
```

Comment:
Please provide a link to download your CSV from the source you got it from. How can I get that CSV from the same source?